app/src/androidTest/java/ch/epfl/favo/MainActivityTest.java

```diff
@@ -1,34 +1,33 @@
-package ch.epfl.favo;
-
-
-import androidx.test.ext.junit.runners.AndroidJUnit4;
-import androidx.test.rule.ActivityTestRule;;
-
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import static androidx.test.espresso.Espresso.onView;
-import static androidx.test.espresso.action.ViewActions.swipeLeft;
-
-import static androidx.test.espresso.assertion.ViewAssertions.matches;
-import static androidx.test.espresso.matcher.ViewMatchers.withId;
-import static androidx.test.espresso.matcher.ViewMatchers.withText;
-
-@RunWith(AndroidJUnit4.class)
-public class MainActivityTest {
-
-    @Rule
-    public final ActivityTestRule<MainActivity> mainActivityActivityTestRule =
-            new ActivityTestRule<>(MainActivity.class);
-    @Test
-    public void testCanChangeTabs(){
-        onView(withId(R.id.text1)).check(matches(withText("1")));
-        //TODO: Replace with actual text in layout
-        onView(withId(R.id.pager)).perform(swipeLeft());
-        onView(withId(R.id.text2)).check(matches(withText("2")));
-        onView(withId(R.id.pager)).perform(swipeLeft());
-        onView(withId(R.id.text3)).check(matches(withText("3")));
-    }
-
-}
+package ch.epfl.favo;
+
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.rule.ActivityTestRule;;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.swipeLeft;
+
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+
+@RunWith(AndroidJUnit4.class)
+public class MainActivityTest {
+
+    @Rule
+    public final ActivityTestRule<MainActivity> mainActivityActivityTestRule =
+            new ActivityTestRule<>(MainActivity.class);
+    @Test
+    public void testCanChangeTabs(){
```

Comment:
Keep in mind that the tests must be updated considering that now entry point of the app is the sign-in page. MainActivityTest in master branch already does so, so when we merge we need to be careful about making tests pass with the new addition